client/src/compontents/login/Login.js

```diff
@@ -1,11 +1,48 @@
-import React,{useState} from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import Form from './../../theme/form';
+import Label from './../../theme/label';
+import Input from './../../theme/input';
+import Button from './../../theme/button';
+
+const H2=styled.h2`
+    text-align:center;`
+
+    const P=styled.p`
+    margin-left:25%;
+    font-size:12px;   
+    &:hover{
+        text-decoration-color:gray;
+        text-decoration:underline;
+        cursor:pointer;
+    }
+    `
+
+    const FormWrap=styled.div`
+    margin-top:30px;
+    margin-left:25%;
+    `
+
+function Login(props) {
+    const [name, setName] = useState('');
```

Comment:
I believe we are using email and password to login.

Reply:
Yes, but after login we still want to get the name information from the backend as well. but I did forget to update the email info.